packages/sysutils/busybox/scripts/init

```diff
@@ -34,6 +34,9 @@
   /bin/busybox mount -t proc proc /proc
   /bin/busybox mount -t sysfs sysfs /sys
 
+  # Just in case... problems with ssh if not correctly owned
+  /bin/busybox chown 0:0 /storage 2>/dev/null
```

Comment:
Sure, I got you - just belt & braces on this one. It's more for consistency with everything else than because it's absolutely required - it does no harm.